Revert #651

aswin-s opened the pull request:
<!-- Add an explanation of the change or anything fishy that is going on -->


This reverts commit 83ae6194b3e4feb363ea9d061085a7ab76e35ffb where we modified the `URL_WEBSITE_REGEX` to prevent excessive backtracking. This however caused regression https://github.com/Expensify/App/issues/37472. So reverting the commit to unblock other PRs. Will raise a new PR with fix.

### Fixed Issues
$ https://github.com/Expensify/App/issues/37472

# Tests
1. What unit/integration tests cover your change? What autoQA tests cover your change?
1. What tests did you perform that validates your changed worked?

# QA
1. What does QA need to do to validate your changes?
1. What areas to they need to test for regressions?

MonilBhavsar commented:
@aswin-s could you please update description to link correct issue "$ GH_LINK"

MonilBhavsar reviewed (commented):
You have mentioned this commit https://github.com/Expensify/expensify-common/commit/17e8f75d879b4a63ec06932cc8fca3e5a6534395
Sorry, I couldn't get the context. The mentioned commit updates babel version

aswin-s commented:
> @aswin-s could you please update description to link correct issue "$ GH_LINK" Also add meaningful PR description And update tests and QA sections

Updated.



> You have mentioned this commit [17e8f75](https://github.com/Expensify/expensify-common/commit/17e8f75d879b4a63ec06932cc8fca3e5a6534395) Sorry, I couldn't get the context. The mentioned commit updates babel version

Fixed the description. Message was auto added by merge commit. Also please hold until I hear back from @AndrewGable or @ntdiary. Not sure if I should revert the PR first or just raise a fix.

MonilBhavsar commented:
@aswin-s you might want to open a PR in App to update expensify-common version
